js-tests/components/ConsortReport_test.js

```diff
@@ -112,4 +112,154 @@ describe('ConsortReport.vue', () => {
       expect(wrapper.vm.showReportForm).toEqual(true);
     });
   });
-});
+
+  describe('drag and drop', () => {
+    let mockReportSummaries;
+
+    beforeEach((done) => {
+      mockReportSummaries = [
+        {
+          id: 'one-id',
+          title: 'Count 1',
+          reportId: 11,
+          strategy: STRATEGY.ITEMIZED,
+          bucketBy: 'some_field_1'
+        },
+        {
+          id: 'two-id',
+          title: 'Count 2',
+          reportId: 22,
+          strategy: STRATEGY.ITEMIZED,
+          bucketBy: 'some_field_2'
+        },
+        {
+          id: 'three-id',
+          title: 'Count 3',
+          reportId: 33,
+          strategy: STRATEGY.TOTAL
+        },
+      ];
+
+      mockProvide = {
+        assetUrls: {},
+        dataService: {
+          fetchReportSummary() {
+            return Promise.resolve(mockReportSummaries);
+          },
+          getReports() {
+            return Promise.resolve([]);
+          },
+          saveReportSummaries(reportSummaries) {
+            return Promise.resolve([]);
+          }
+        }
+      };
+
+      wrapper = shallowMount(ConsortReport, {
+        provide: mockProvide
+      });
+
+      wrapper.vm.configPromise.then(() => done());
+    });
+
+    it('default dnd state', () => {
+      const { dragItemId, dropItemId, startIndex, originalArray } = wrapper.vm.dndState;
+      expect(dragItemId).toEqual(null);
+      expect(dropItemId).toEqual(null);
+      expect(startIndex).toEqual(null);
+      expect(originalArray).toEqual(null);
+    });
+
+    it('summary is not being dragged', () => {
```

Comment:
I think here you are asserting that the summary IS being dragged.

Reply:
Oh good catch.